hydra_agent/querying_mechanism.py

```diff
@@ -672,22 +672,20 @@ def main():
     Take URL as an input and make graph using initilize function.
     :return: call query function for more query.
     """
-    url = input("url>>>").strip()
+    url = input("url>>>").strip().rstrip('/')
```

Comment:
this is still using the CLI input, the function has to be made generic and eventually `click` has to be added for the CLI usage (we can add CLI functionalities later, now focus on generic functions)